ocaml-lsp-server/src/code_actions/action_remove_unused.ml

```diff
@@ -0,0 +1,142 @@
+open Import
+
+let action_title = "Remove unused"
+
+let slice doc (range : Range.t) =
+  let src = Document.source doc in
+  let (`Offset start) = Msource.get_offset src @@ Position.logical range.start
+  and (`Offset end_) = Msource.get_offset src @@ Position.logical range.end_ in
+  String.sub (Msource.text src) ~pos:start ~len:(end_ - start)
```

Comment:
String.drop?

Reply:
Doesn't that just drop the start of the string?